README.md

```diff
@@ -60,24 +64,30 @@ Macros are also supported for text and primitives:
 
 ```rust
 use embedded_graphics::prelude::*;
-use embedded_graphics::{circle, icoord, line, rect, text_6x8, triangle};
+use embedded_graphics::{egcircle, icoord, egline, egrectangle, text_6x8, egtriangle};
+use embedded_graphics::pixelcolor::BinaryColor::On as C1;
+use embedded_graphics::pixelcolor::BinaryColor::Off as C0;
+
+// Only used for examples - this would be replaced by the driver for your chosen display
+use embedded_graphics::mock_display::MockDisplay as Display;
 
 fn main() {
     // Create a display object to draw into
     // This will be whichever display driver you decide to use, like the SSD1306, SSD1351, etc
     let mut display = Display::new();
 
-    display.draw(egcircle!((64, 64), 64, stroke = Some(1u8)));
-    display.draw(egline!((64, 64), (0, 64), stroke = Some(1u8)));
-    display.draw(egline!((64, 64), (80, 80), stroke = Some(1u8)));
-    display.draw(egrectangle!((64, 64), (80, 80), stroke = None, fill = Some(2u8)));
-    display.draw(text_6x8!("Hello world!", stroke = Some(1u8)).translate(icoord!(5, 50)));
+    display.draw(egcircle!((64, 64), 64, stroke = Some(C1)));
+    display.draw(egline!((64, 64), (0, 64), stroke = Some(C1)));
+    display.draw(egline!((64, 64), (80, 80), stroke = Some(C1)));
+    display.draw(egrectangle!((64, 64), (80, 80), stroke = None, fill = Some(C0)));
+    display.draw(text_6x8!("Hello world!", stroke = Some(C1)).translate(icoord!(5, 50)));
 }
 ```
 
 ## Cargo Features
 
-* `nalgebra_support` - use the [Nalgebra](https://crates.io/crates/nalgebra) crate with `no_std` support to use as the `Coord` type. This should allow you to use most Nalgebra methods on objects rendered by embedded_graphics.
+* `nalgebra_support` - use the [Nalgebra](https://crates.io/crates/nalgebra) crate with `no_std`
+support to enable conversions from `nalgebra::Vector2` to [`Coord`] and [`UnsignedCoord`].
```

Comment:
``[`Coord`] and [`UnsignedCoord`]`` doesn't render correctly outside of rustdoc

Reply:
Oops, thanks for spotting that. Just fixed it.